docs/user-guide/kubeconfig-file.md

```diff
@@ -94,7 +94,89 @@ users:
     client-key: path/to/my/client/key
 ```
 
-### Building your own kubeconfig file
+### Breakdown/explanation of components
+
+#### cluster
+
+```
+clusters:
+- cluster:
+    api-version: v1
+    server: http://cow.org:8080
+  name: cow-cluster
+```
+
+A `cluster` contains endpoint data for a kubernetes cluster. This includes the fully
+qualified url for the kubernetes apiserver, as well as the cluster's certificate
+authority or `insecure-skip-tls-verify: true`, if the cluster does not support tls.
```

Comment:
"if the cluster does not support tls." isn't quite right.  "If the cluster's serving certificate is not signed by a system trusted certificate authority."?